### ISSUE TITLE
Modification of YMAP entity indexing

### ISSUE THREAD

**Issue by Gusepe** [Sollumz/Sollumz#945]
When importing/exporting a YMAP, sollumz does not keep the same indexing.
This makes it impossible to edit LOD ymaps, since the indexing must be identical to the index entered in each child.

**Scenario 1:**

I want to edit a lod vanila YMAP, which will break all the children's indexing.

**Scenario 2:**

I create my child and parent ymaps, and if I add a new drawable to my lod YMAP, I'll have to redo all my children's indexing if my new lod drawable is not positioned last in my outliner.

![image](https://github.com/user-attachments/assets/ec41b9dc-2c2b-4a0e-a27b-204396d2b331)

**Comment by Prompt-Coder** [comment]
Yeah it seems totally random currently, struggle with this as well. 

The great thing would be to have a working LODS system with hierarchy menu and etc, so even if they are random every time indexis are calculated on the go.
